numpy/random/generator.pyx

```diff
@@ -3921,7 +3921,7 @@ cdef class Generator:
         Randomly permute a sequence, or return a permuted range.
 
         If `x` is a multi-dimensional array, it is only shuffled along its
-        first index.
+        first index. 
```

Comment:
Please revert the whitespace additions

Reply:
The spurious space is still there.

Reply:
Still there.  @maxwell-aladago, the space added at the end of this line should not be there.